lib/profiles/geojson.json

```diff
@@ -1,16 +1,16 @@
 {
     "$schema": "http://json-schema.org/draft-04/schema#",
-    "id": "https://raw.githubusercontent.com/fge/sample-json-schemas/master/geojson/geojson.json#",
+    "id": "http://json-schema.org/geojson/geojson.json#",
```

Comment:
@georgiana-b 
If some of types are blockers please just skip in tests it for now and create an issue.